src/components/Swap/Summary/index.tsx

```diff
@@ -64,7 +64,7 @@ interface EstimateProps {
   trade: InterfaceTrade
 }
 
-function Estimate({ trade, slippage }: EstimateProps) {
+export function Estimate({ trade, slippage }: EstimateProps) {
```

Comment:
since this is exported now maybe name this something more specific, like `SwapEstimate` 🤔